versioned_docs/version-8.1/components/zeebe/zeebe-overview.md

```diff
@@ -15,7 +15,7 @@ With Zeebe you can:
 - Use as part of a software as a service (SaaS) offering with Camunda 8 or deploy with Docker and Kubernetes (in the cloud or on-premises) with Camunda 8 Self-Managed.
 - Scale horizontally to handle very high throughput.
 - Rely on fault tolerance and high availability for your processes.
-- Export processes data for monitoring and analysis (currently only available through the [Elasticsearch exporter](https://github.com/camunda-cloud/zeebe/tree/develop/exporters/elasticsearch-exporter) added in Camunda 8 Self-Managed).
+- Export processes data for monitoring and analysis (currently only available through the [Elasticsearch exporter](https://github.com/camunda/zeebe/tree/stable/8.1/exporters/elasticsearch-exporter) added in Camunda 8 Self-Managed).
```

Comment:
The `elasticsearch-exporter` docs where not part of this repo at that version yet